### DISCUSSION TITLE
Is there any tutorials for ASAP7 + OpenLane?

### DISCUSSION THREAD

**Discussion by armleo** [The-OpenROAD-Project/OpenLane#1055] [General] [answered]
Is there any tutorials for ASAP7 + OpenLane?

**Comment by maliberty** [comment, marked as answer]
Not really as it is a work in progress.  For now I would suggest using asap7 through openroad-flow-scripts.  If you really want to try it out I could write some quick directions.

**Comment by armleo** [comment]
Yeah, I think that would help to get started with ASAP7 + OpenRoad. I found some documentation on OpenRoad, but I still cant figure out on how to make OpenRoad to use asap7.

**Reply by maliberty** [reply]
You would use openroad-flow-scripts, something like:
```
DESIGN_CONFIG=./designs/asap7/gcd/config.mk make
```
from the flow/ dir
